client/Components/Dogs/dogs.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../variables.scss';
+#dogs {
+    margin:10% auto;
+    display: flex;
```

Comment:
we may want to add `flex-display: wrap`. right now with more than 3 dogs, it looks like this 
![Uploading image.png…]()

Reply:
truee